web/yo/app/scripts/controllers/gene.js

```diff
@@ -2514,17 +2514,78 @@ angular.module('oncokbApp')
                 mutationEffect.addOn.setText('');
             };
 
-            $scope.move = function(driveList, index, moveIndex, event) {
+            $scope.displayAngle = function (parent, item, type) {
+                var index = parent.indexOf(item);
```

Comment:
Let's do a quick check on parent and item. Return false if the type is incorrect